docs/guides/rtc_product_guide.md

```diff
@@ -89,7 +89,7 @@ When custom-ordering imagery, however, the DEM Matching option is available for
 
 During processing, a surface scattering area image for the scene is calculated and saved. This projected area image is used to create the RTC product - the SAR image is multiplied by the ratio of an ellipsoidal scattering image (used during calibration) and this scattering area image. Note that this image is, by default, projected to gamma-nought (γ<sub>0</sub>). However, when custom ordering imagery,  selecting sigma-nought (σ<sub>0</sub>) output will result in a sigma-nought projected area image.  
```

Comment:
@talogan I thought that the scattering image was sigma-0 by default, but this description has the opposite. Thought I'd check to make sure.

Reply:
I checked the code and my emails.  The scattering area image is actually gamma0.  There is no sigma0 scattering area image calculated right now.  

Instead, GAMMA's pixel_area program creates
- **gamma0_pix** (scattering area image saved and the simulated image used during co-registration)
- **sigma_ratio image** (which converts ellipsoidal radiometrically calibrated sigma0 into DEM-based sigma0)
- **sig2gam_ratio image** (which converts ellipsoidal radiometrically calibrated sigma0 into DEM-based  gamma0)

During processing, the appropriate one of these ratio images is multiplied by the sigma0 calibrated backscatter to get either DEM-based sigma0 or DEM-based gamma0.

Reply:
OK, thanks for checking that out.